CHANGELOG.md

```diff
@@ -87,7 +98,6 @@
 * [ENHANCEMENT] Ingester: improve performance when Active Series Tracker is in use. #4717
 * [ENHANCEMENT] Store-gateway: optionally select `-blocks-storage.bucket-store.series-selection-strategy`, which can limit the impact of large posting lists (when many series share the same label name and value). #4667 #4695 #4698
 * [ENHANCEMENT] Querier: Cache the converted float histogram from chunk iterator, hence there is no need to lookup chunk every time to get the converted float histogram. #4684
-* [ENHANCEMENT] Improved memory limit on the in-memory cache used for regular expression matchers. #4751
```

Comment:
We should remove `main / unreleased` section from CHANGELOG in this PR.

PRs that were cherry-picked from `main` to `release-2.8` should have entries in the 2.8 section, but that should be fixed in a separate PR, not this one.

Reply:
This specific entry:

```
* [ENHANCEMENT] Improved memory limit on the in-memory cache used for regular expression matchers. #4751
```

should be kept in 2.8 section, without modifications.

Reply:
I think it would make most sense to add a `2.8.0-rc.1` section on top and have this in it:
```
## 2.8.0-rc.1

### Grafana Mimir

* [ENHANCEMENT] Improved memory limit on the in-memory cache used for regular expression matchers. #4751
* [ENHANCEMENT] Go: update to 1.20.3. #4773
```

Upon release we merge to a 2.8.0 section and reconcile with main on merge to main. @pstibrany wdyt?

Reply:
I'm also ok with renaming `2.8.0-rc.0` section to `2.8.0-rc.1`  and having the above mentioned two items merged in there. Just let's not lose track of items.

Reply:
I will just rename the rc.0 section to rc.1 and put the new items from backport there for now.

Reply:
ok, but then removal of  line 90 isn't backporting.

Reply:
We don't rename rc.0, but we can add rc.1.

Reply:
also you're missing the new changelog entry from the backport

Reply:
I agree with @krajorama suggestion.

Reply:
> ok, but then removal of line 90 isn't backporting.

 I added it to the first backport I created. I intended to add all backport changelog in the different PR.  
 
 > also you're missing the new changelog entry from the backport
 
 Yes I intended to add the changelog after rc.0 in a new PR.
 
 All in all, I will just add new rc1. section and get this merged. I will document this process too.